defaults/main.yml

```diff
@@ -14,6 +14,8 @@ enduro_listen_temporal_address: "127.0.0.1:7233"
 enduro_api_debug_enabled: "false"
 enduro_create_pipeline_transfer_dir: "false"
 enduro_create_system_user: "true"
+systemd_environment_path: "{% if ansible_os_family == 'RedHat' %}/etc/sysconfig{% else %}/etc/default{% endif %}"
```

Comment:
Could this just be a default without the `RedHat` check and be overwritten by vars/rocky and others?